frontend/src/components/projectDetail/questionsAndComments.js

```diff
@@ -170,12 +159,12 @@ export function CommentList({ userCanEditProject, projectId, comments, retryFn }
             </div>
             <div>
               {(userCanEditProject || comment.username === username) && (
-                <DeleteButton
-                  className={`bg-transparent bw0 w2 h2 lh-copy overflow-hidden blue-light p0 mb1 hover-red`}
-                  showText={false}
-                  onClick={() => {
-                    deleteComment(comment.id);
-                  }}
+                <DeleteModal
+                  id={comment.id}
+                  type={'comments'}
+                  className="bg-transparent bw0 w2 h2 lh-copy overflow-hidden blue-light p0 mb1 hover-red"
+                  onDelete={retryFn}
+                  endpointURL={`projects/${projectId}/comments/${comment.id}/`}
                 />
```

Comment:
It seems that the `showText` props might have been accidentally omitted when passing them to the `DeleteButton` component within the `DeleteModal`.